tests/Config/Guesser/SourceDirGuesserTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Infection\Tests\Config\Guesser;
+
+use Infection\Config\Guesser\SourceDirGuesser;
+use PHPUnit\Framework\TestCase;
+
+class SourceDirGuesserTest extends TestCase
```

Comment:
Just found another `SourceDirGuesserTest` file at the incorrect path (`src/tests/Guesser`).

Could you please move tests from that file to your file and remove it?

Reply:
@borNfreee done